runhouse/main.py

```diff
@@ -783,11 +794,14 @@ def restart(
     )
 
 
-@app.command()
+@server_app.command()
 def stop(
     stop_ray: bool = typer.Option(False, help="Stop the Ray runtime"),
     cleanup_actors: bool = typer.Option(True, help="Kill all Ray actors"),
 ):
+    """
+    Stopping Runhouse server on the current cluster.
```

Comment:
this doesn't have to be on the current cluster right? can we support this + the other server commands locally by specifying a cluster name?

Reply:
Yeah. So the `restart` and `stop` are supported for both locally on the user laptop and on the cluster.

Regarding the start cmd - not sure how will it differ from restart if we support if both ways. I think we should discuss it further.